vars/virtualenv.groovy

```diff
@@ -1,16 +1,8 @@
-def create(String python, String destDir) {
-  sh("virtualenv --python=${python} ${destDir}")
-}
-
-
-def installRequirements(String destDir) {
-  run(destDir, "pip install -r requirements.txt")
-}
+import com.ableton.VirtualEnv
 
 
-def run(String destDir, String command) {
-  sh("""
-    . ./${destDir}/bin/activate
-    ${command}
-  """)
+def create(script, String python) {
+  venv = new VirtualEnv(script, python)
+  script.sh("virtualenv --python=${python} ${venv.destDir}")
```

Comment:
Last comment :smile:
`venv.script.sh()`?

Reply:
Technically it doesn't matter, but sure, I can do that.